includes/modules/meta_tags.php

```diff
@@ -22,6 +22,8 @@
 } else {
   define('TAGLINE', '');
 }
+// array to remove specific characters from meta-data
+$bad_characters = array('"','*');
```

Comment:
Can you move this initialization of `$bad_characters` up to line 16 (replace the comment line of "this should be the first line" with your lines 25-26.

Then add `$bad_characters` to the list of params passed to the notifier on line 17 above. This way people can extend the list by using an observer class.